x/plans/types/geolocation.go

```diff
@@ -71,9 +73,16 @@ func GetAllGeolocations() []int32 {
 func GetGeolocationsFromUint(geoloc int32) []Geolocation {
 	geoList := []Geolocation{}
 	allGeos := GetAllGeolocations()
-	for _, geo := range allGeos {
-		if geo&geoloc != 0 {
-			geoList = append(geoList, Geolocation(geo))
+
+	if geoloc == int32(Geolocation_GL) {
+		for i := 0; i < len(allGeos); i++ {
```

Comment:
return the list from the init

Reply:
I can't because the function returns []Geolocation and not []int32. Should I change it?

Reply:
lets be consistent with this, we either work with array of list or enum, whatever you choose

Reply:
done. see https://github.com/lavanet/lava/commit/29385ec174e0adc153f82be836b3e7849f78d8df